packages/thirdweb/src/wallets/in-app/core/authentication/guest.ts

```diff
@@ -1,37 +1,60 @@
+import type { AsyncStorage } from "src/utils/storage/AsyncStorage.js";
+import { generateRandomUint192 } from "src/wallets/smart/lib/utils.js";
```

Comment:
nope, relative imports. please make sure ur IDE is setup correctly it should've done this FOR you